### DISCUSSION TITLE
AU - Layers names

### DISCUSSION THREAD

**Discussion by bfrichet3** [INSPIRE-MIF/helpdesk#88] [Q&A] [answered]
Dear Sir or Madame

I hope you are all right.  I am working with the conformity of our WMS with the INSPIRE TG. 

I noticed in the [TG concerning AU](https://inspire.ec.europa.eu/id/document/tg/au) two points of interest concerning the conformity of our WMS.

1) The [regulation 1089/2010] article 14 3.   states that "for spatial object types whose objects can be further classified using a code list-valued attribute, several layers may be defined". As I read it, we can deliver one separated layer for each administrative level for both AdminstrativeUnit and AdministrativeBoundary types (both feature types include AdministrativeHierarchyLevel as a code-list);

2) The names of these multiple layers are speciefied by annex II of the same [regulation 1089/2010]. If we publish one layer for each administrative level, layer names should be like that AU.<CodelistValue> where <CodelistValue> is equal to the concerned code from AdministrativeHierarchyLevel codelist (e.g. 1stOrder) for both AdministrativeUnit and AdminstrativeBoundary.

![image](https://user-images.githubusercontent.com/80756452/162975041-e82d58d9-9b3a-44fd-917f-709a9681ab43.png)

So this is my question: following these requirements, how could we create layers for each administrative level and be compliant with OGC WMS regulation (each name should be unique).

Regards,

Benoît

**Comment by sMorrone** [comment, marked as answer]
Dear @bfrichet3 ,
with reference to the third table column “**Spatial object type(s)**”, the obligation to provide layer Name according to **`AU.<CodelistValue>`** encoding, is referred to spatial object type **MaritimeZone** and specifically to the _MaritimeZoneTypeValue_ codelist. This becomes clearer when reading the table at the end of the page listing “_the layers for all code list values_” and /or querying the [INSPIRE layer register ](https://inspire.ec.europa.eu/layer) for Administrative Units layers.
In case one wants to provide more layers for the AdministrativeUnit and the AdminstrativeBoundary spatial objects following classification in the _AdministrativeHierarchyLevel_ code list, the `AU.<CodelistValue>`  nomenclature is not mandatory. One could e.g., use `AU.AdministrativeUnit.<CodelistValue>` and `AU.AdministrativeBoundary.<CodelistValue>`.
Warm regards,
Stefania

**Comment by bfrichet3** [comment]
Dear @sMorrone 

Thank you for you quick answer. I will follow you recommandation. Neverthless I think that the specifications aren't explicit enought on that matter. Maybe we should clarify them (otherwise, they are really understandable).

Regards,

**Comment by sMorrone** [comment]
Since the purpose of the the technical guidelines repository is the management of the change proposals to the TGs, I am moving this issue to the [general helpdesk ](https://github.com/INSPIRE-MIF/helpdesk)repository. More specifically, I will move it to the "Discussions" tab and classify it as "[Q&A](https://github.com/INSPIRE-MIF/helpdesk/discussions/categories/q-a)"
